app/models/hyrax/base.rb

```diff
@@ -0,0 +1,11 @@
+module Hyrax
+  class Base
```

Comment:
does it make sense for this to live in a `Base` class? What about pushing this into a search service?

I think it would be worth looking about in `services/hyrax` to see if there's already a natural home for this functionality.

Reply:
Alternatively, both existing calls seem like they're weird optimizations, designed to avoid loading data from fedora. I wonder if we couldn't refactor them away?

Reply:
@no-reply I've moved it to Hyrax::SolrService class since the behavior of the method is related to SOLR search. And yes, it seems like something that we could refactor at some point later.